packages/common/src/utilities/string/index.ts

```diff
@@ -95,22 +108,47 @@ export function stringifyPlusPlus(object: any): string {
     return 'undefined';
   }
 
-  // Don't JSON.stringify strings, because we don't want quotes in the output
+  // Don't JSON.stringify strings, because might not want quotes in the output
   if (typeof object === 'string') {
-    return object;
+    return options.quoteStrings ? `"${object}"` : object;
+  }
+
+  if (Array.isArray(object)) {
+    if (isEachObjectAPrimitiveType(object)) {
+      return (
```

Comment:
I think these could be more cleanly written with template strings and maybe some variables.

Reply:
Maybe the top one.  But the bottom one is best kept as is because of newlines, which are really hard to reason about in a template string.  And in that case, if I keep the bottom one as is, it makes sense to keep the top one as is as well.  So -- duly noted, but I don't think I have a clean way of fixing this.